plugins/UM3NetworkPrinting/src/ClusterUM3OutputDevice.py

```diff
@@ -406,6 +407,11 @@ def deleteJobFromQueue(self, print_job_uuid: str) -> None:
         # is a modification of the cluster queue and not of the actual job.
         self.delete("print_jobs/{uuid}".format(uuid = print_job_uuid), on_finished=None)
 
+    @pyqtSlot(str)
+    def forceSendJob(self, print_job_uuid: str) -> None:
```

Comment:
See my previous comment about removing the config changes and switch to warning. If you change that, you can actually add a overrideWarning function to the printJobOutputModel, which can then be implemented by the configurationOutputController to send this.